<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="RigExport_Tab">
     <attribute name="title">
      <string>Rig Export</string>
     </attribute>
     <widget class="QPushButton" name="createHierachy_Button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Layer Hierachy</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Beard_RadioButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>52</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Beard</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Face_RadioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>52</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Face Rig</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Hair_RadioButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>52</y>
        <width>57</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hair</string>
      </property>
     </widget>
     <widget class="QComboBox" name="rigGroup_comboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rig Group</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="FullRig_RadioButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>54</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Full Rig</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AnimExport_Tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Animation Export</string>
     </attribute>
     <widget class="QCheckBox" name="facialCheckBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>With Facial Animation</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>191</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="refImport_Button">
         <property name="text">
          <string>Import Reference</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bakeAnim_Button">
         <property name="text">
          <string>Bake Animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="extractLayers_Button">
         <property name="text">
          <string>Create Hierachy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
